index.bs

```diff
@@ -4590,3 +4763,14 @@ Possible mitigations:
     presence from the reporting origin. Compared to the previous mitigation, the
     proxy server could itself handle the [=event-level report/trigger priority=]
     functionality, at the cost of increased complexity in the proxy.
+
+## Attribution Scope ## {#attribution-scope}
```

Comment:
Let's add an `ISSUE` to https://wicg.github.io/attribution-reporting-api/#navigation-monkeypatches linking to this section.